integrations/integration_test.go

```diff
@@ -96,11 +99,14 @@ func initIntegrationTest() {
 		}
 		defer rows.Close()
 
-		if !rows.Next() {
-			if _, err = db.Exec("CREATE DATABASE testgitea"); err != nil {
-				log.Fatalf("db.Exec: %v", err)
+		if rows.Next() {
+			if _, err = db.Exec("DROP DATABASE testgitea"); err != nil {
```

Comment:
When run on drone it's no need. But when run `make test-mysql` local, it will failed when you run twice because the old data didn't cleanup. This change will drop the old database and load data from *.yml file.